graphql_server/server.go

```diff
@@ -13,9 +14,10 @@ import (
 
 // Server is our graphql Server.
 type Server struct {
-	Users []*data.User
-	Games []*data.Game
-	Runs  []*data.Run
+	Users  []*data.User
+	Games  []*data.Game
+	Runs   []*data.Run
+	Logger zerolog.Logger
```

Comment:
Synced offline and came up with a solution that will use a shared logging package with a globally exported member to avoid the future plumbing problem with this approach.